spec/requests/purchases_requests_spec.rb

```diff
@@ -264,7 +264,20 @@
 
     describe "GET #show" do
       let(:item) { create(:item) }
-      let!(:purchase) { create(:purchase, :with_items, item: item) }
+      let(:storage_location) { create(:storage_location, organization: organization, name: 'Some Storage') }
+      let(:vendor) { create(:vendor, organization: organization, business_name: 'Another Business') }
+      let!(:purchase) { create(:purchase, :with_items, item: item, storage_location: storage_location, comment: 'Fine day for diapers, it is.') }
+
+      it "shows the purchase info" do
+        escaped_html_comment = CGI.escapeHTML(purchase.comment_view)
```

Comment:
You know exactly what the comment is - please use that value in your expectation instead of referencing the `purchase`record.

Reply:
Made the changes here [83ee754](https://github.com/rubyforgood/human-essentials/pull/4642/commits/83ee754272535324c78998a8c4b6165c5ac2e986)